lib/rule-tester/rule-tester.js

```diff
@@ -705,6 +726,7 @@ class RuleTester {
                 assert(Array.isArray(item.options), "options must be an array");
                 ruleConfig = [1, ...item.options];
             }
+            runHook(item, "before");
```

Comment:
maybe I should move it closer to `runHook(..., "after")` for better clarity...

Reply:
addressed in [c557112](https://github.com/eslint/eslint/pull/18771/commits/c5571129b1d2e673c57a7a4c8b5b897422091077)